src/components/presentational/AdminDashboard/styles.ts

```diff
@@ -3,5 +3,18 @@ import { makeStyles } from '@material-ui/core/styles';
 export const createSessionStyle = makeStyles(theme => ({
     lockup: {
         textAlign: 'center'
-    }
+    },
```

Comment:
Do we need this ,?

Reply:
nope, but its not a problem having it there :p